### PR TITLE
updated-pr-data

### PR DESCRIPTION
**Pull request by tt0suzy**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -3133,7 +3133,7 @@ https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleSslSocketFactory,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleVerifier,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.verifierAccepts,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
-https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
+https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,MovedOrRenamed,,https://github.com/kiegroup/optaplanner/commit/4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69
```
Comment (darko-marinov) [inline pr-data.csv:3136]: What script did you use to find that the test was MovedOrRenamed? Where is the new location? README says to add a new location for each such test.

Comment (tt0suzy) [reply pr-data.csv:3136]: It's the module that got renamed not the test though, if you think this change should not be considered as MovedOrRenamed I can remove it.

Comment (tt0suzy) [reply pr-data.csv:3136]: Updated, also the test `com.spotify.dbeam.args.JdbcExportArgsTest.should parse correctly for mysql connection` is deleted. Neither test file or test suite exists now.

File: pr-data.csv [hunk 2]
```diff
@@ -3696,7 +3697,7 @@ https://github.com/spinn3r/noxy,d53a49421f385c70b5abe7e8cda84ff3a7b59c71,noxy-di
 https://github.com/spinn3r/noxy,d53a49421f385c70b5abe7e8cda84ff3a7b59c71,noxy-discovery-zookeeper,com.spinn3r.noxy.discovery.zookeeper.ZKTest.testMembershipJoinAndLeave,NDOD;UD,,,
 https://github.com/spinn3r/noxy,d53a49421f385c70b5abe7e8cda84ff3a7b59c71,noxy-reverse,com.spinn3r.noxy.reverse.ReverseProxyServiceTest.testRequestMetaForSuccessfulRequest,ID,Opened,https://github.com/spinn3r/noxy/pull/21,
 https://github.com/spotify/apollo,a03105ee1f98f8c5878f4e3c4d03c8732c6384bd,apollo-api-impl,com.spotify.apollo.route.MiddlewaresTest.asShouldSerializeResponseObjectAsJson,ID,RepoArchived,,
-https://github.com/spotify/dbeam,70837e8f610c16c012be424341857236d4242785,dbeam-core,com.spotify.dbeam.args.JdbcExportArgsTest.should parse correctly for mysql connection,UD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/33
+https://github.com/spotify/dbeam,70837e8f610c16c012be424341857236d4242785,dbeam-core,com.spotify.dbeam.args.JdbcExportArgsTest.should parse correctly for mysql connection,UD,Deleted,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/33
```
Comment (darko-marinov) [inline pr-data.csv:3700]: Where was this deleted? Note a pending PR with a script for checking where tests were deleted.

File: pr-data.csv [hunk 3]
```diff
@@ -4120,7 +4121,7 @@ https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30a
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase9,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
 https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,RepoArchived
 https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToXML,ID,RepoArchived,,
-https://github.com/twitter/cloudhopper-commons,05f06af02facfe4414c2ceaa799182c6dffbc1b4,ch-commons-util,com.cloudhopper.commons.util.MetaFieldUtilTest.toMetaFieldInfoArray,ID,,,
+https://github.com/twitter/cloudhopper-commons,05f06af02facfe4414c2ceaa799182c6dffbc1b4,ch-commons-util,com.cloudhopper.commons.util.MetaFieldUtilTest.toMetaFieldInfoArray,ID,Opened,https://github.com/twitter/cloudhopper-commons/pull/26,
```
Comment (darko-marinov) [inline pr-data.csv:4124]: Please follow up on CLA.

File: pr-data.csv [hunk 4]
```diff
@@ -3133,7 +3133,8 @@ https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6
...
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleVerifier,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.verifierAccepts,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
-https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
+https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,MovedOrRenamed,,https://github.com/kiegroup/optaplanner/commit/4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69
+https://github.com/kiegroup/optaplanner,4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69,core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:3137]: Did you confirm that this was still ID in that commit `4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69`? Did you try it in the latest commit?

Comment (tt0suzy) [reply pr-data.csv]: I removed the 2 rows of non-pr update for now, I'll work on that later. Now this pr only contains pr updates.

**Comment by darko-marinov** [comment]
Good idea to split the PR. I accepted the current changes, and you can work on the rest later.

**Pull request merged**